src/Stripe.net/Services/Accounts/AccountCapabilitiesOptions.cs

```diff
@@ -0,0 +1,61 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class AccountCapabilitiesOptions : INestedOptions
+    {
+        /// <summary>
+        /// The au_becs_debit_payments capability.
```

Comment:
We don't use snake case in docstrings in dotnet usually and would write `The AU BECS Debit payments capability` usually